foreshadow/intents/numeric.py

```diff
@@ -45,3 +47,31 @@ def transform(self, X, y=None):
 
         """
         return X.apply(pd.to_numeric, errors="coerce")
+
+    @classmethod
+    def column_summary(cls, df):  # noqa
+        data = df.iloc[:, 0]
+        nan_num = int(data.isnull().sum())
+        invalid_num = int(
+            pd.to_numeric(df.iloc[:, 0], errors="coerce").isnull().sum()
```

Comment:
why don't we use data instead of df.iloc[:, 0] here?

Reply:
Not sure. This was copied from the V0.1.0 code. Your point is valid. If the intent is only applied to a single column, using the index selection is unnecessary. Let me verify that.

Reply:
`to_numeric` requires a list, tuple, 1-d array, or Series. Otherwise it will fail so we couldn't use the df directly.